package.json

```diff
@@ -134,55 +114,76 @@
     ]
   },
   "scripts": {
+    "precommit": "pretty-quick --staged",
+    "prepush": "tsc -p browser/tsconfig.json && yarn lint",
```

Comment:
Seems like this is only build the browser - would it make sense to run `npm run build && npm run lint` to validate all the projects we build?

Reply:
@bryphe great suggestion I was just testing out commands at that point but those would be good commands to run there wasn't sure which to use